test/test.transporters.js

```diff
@@ -71,6 +71,17 @@ describe('Transporters', function() {
     assert.equal(req.body, null);
   });
 
+it('should return errors as instances of Error', function (done) {
+    var google = require('../lib/googleapis');
+    var pagespeed = google.pagespeedonline('v1');
+
+    // intentional grammar mistake to trigger the error
+    pagespeed.pagespeedapi.runpagespeed({url: 'http://gogle.con'}, function (err) {
```

Comment:
Actually, how would you simulate a DNS resolution failure? *Nock* sure can't do it. Also, recently I gave [this answer](http://stackoverflow.com/questions/27708960/how-to-test-error-in-request-with-nock/27765293#27765293) over on SO to exactly this kind of problem and I am not sure if I want to go into such monkey-patching in real world...:)

So, maybe to clarify - this test does not try to prevent the network call from happening, but in reality it will never happen because it cannot resolve *http://gogle.con* into IP address.